tests/env/test_explicit_environment.py

```diff
@@ -1,108 +0,0 @@
-# Copyright (C) 2012 Anaconda, Inc
```

Comment:
We don't want to support an explicit environment.yaml anymore. The `conda.env.env.Environment` class should just represent the conda environment.yaml format.